src/documents/consumer.py

```diff
@@ -691,6 +691,8 @@ def _parse_title_placeholders(self, title: str) -> str:
             added_month_name_short=local_added.strftime("%b"),
             added_day=local_added.strftime("%d"),
             owner_username=owner_username,
+            original_filename=Path(self.filename).with_suffix("").name,
```

Comment:
You can actually just use [`.stem`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.stem) here

Reply:
The code I got previously is from code in "file_handling.py" line 190.

I also just tested **.stem** and it works. So, I changed to **.stem** now.

Thanks,